### PR TITLE
[15.0][MIG] hr_employee_calendar_planning: Migration to 15.0

### PR DESCRIPTION
**Pull request by victoralmau**
Superseed https://github.com/OCA/hr/pull/1086

Migration to 15.0

Please @pedrobaeza and @CarlosRoca13 can you review it?

@Tecnativa TT36463

### CODE REVIEW

File: hr_employee_calendar_planning/readme/CONFIGURE.rst [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#. Go to *Employees > Employees*.
+#. Open or create a new one.
+#. On the "Work Information" tab, fill the section "Calendar planning" with:
```
Comment (CarlosRoca13) [inline hr_employee_calendar_planning/readme/CONFIGURE.rst]:
```suggestion
#. On the "Work Information" tab, fill the section "Working Hours" with:
```

**Review by CarlosRoca13 — APPROVED** [review]

File: hr_employee_calendar_planning/i18n/es.po [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+"otra compañía."
+
+#. module: hr_employee_calendar_planning
+#: code:addons/hr_employee_calendar_planning/models/resource_calendar.py:0
+#, python-format
+msgid ""
+"%(item_name)s is used in %(total_items)s employee(s).You should change them "
+"first."
+msgstr ""
+"%(item_name)s está usado en %(total_items)s empleado(s). Deberías cambiarlo "
```
Comment (pedrobaeza) [inline hr_employee_calendar_planning/i18n/es.po]: Always "trato de usted"

```suggestion
"%(item_name)s está usado en %(total_items)s empleado(s). Debería cambiarlo "
```

File: hr_employee_calendar_planning/i18n/es.po [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+"X-Generator: Poedit 3.0.1\n"
+
+#. module: hr_employee_calendar_planning
+#: code:addons/hr_employee_calendar_planning/models/resource_calendar.py:0
+#, python-format
+msgid ""
+"%(item_name)s is used in %(total_items)s employee(s) related to another "
+"company."
+msgstr ""
+"%(item_name)s está usado en %(total_items)s  empleado(s) relacionado con "
```
Comment (pedrobaeza) [inline hr_employee_calendar_planning/i18n/es.po]: Let's be inclusive and complete:

```suggestion
"%(item_name)s está usado en %(total_items)s empleado/a(s) relacionado/a(s) con "
```

**Comment by victoralmau** [comment]
Changes done.

**Review by pedrobaeza — APPROVED** [review]

File: hr_employee_calendar_planning/views/hr_employee_views.xml [hunk 4]
```diff
@@ -10,7 +10,10 @@
                 <attribute name="invisible">1</attribute>
             </field>
             <field name="resource_calendar_id" position="after">
-                <field name="calendar_ids" context="{'default_company_id': company_id}">
+                <field
+                    name="calendar_ids"
+                    context="{'default_company_id': parent.company_id}"
```
Comment (etobella) [inline hr_employee_calendar_planning/views/hr_employee_views.xml]: Why the parent? It was removed on other PRs company_id is already a field from hr.employee so it should not be necessary.

You can check that it does not work following this steps:

- Create a company
- Select both companies on your view
- Create a new employee for company B and create a calendar from the widget. The calendar created will be from company A not B.

Comment (victoralmau) [reply hr_employee_calendar_planning/views/hr_employee_views.xml]: Sorry, following the steps you indicate I see that it was not working correctly.

The change that I have applied work fine, right?
ejemplo-v15

**Review by etobella — APPROVED** [review]
Yeah!!! Now seems to work fine! Thanks! :smile:

**Pull request merged**